internal/preview/preview.go

```
@@ -67,42 +66,58 @@ func Create(params CreateParams) error {
-	if err := patch.SetPath(node, []string{"spec", "version"}, patch.Scalar(params.Version)); err != nil {
-		return fmt.Errorf("setting spec.version: %w", err)
+	source.Name = target
+	source.Spec.Version = params.Version
+	if source.Labels == nil {
+		source.Labels = map[string]string{}
 	}
+	source.Labels[v1alpha1.PreviewLabel] = "true"
+
+	text, err := func() ([]byte, error) {
+		tree := yml.Clone(source.File.Tree)
+		yml.SetOrAddNodeValue(tree, "metadata.name", target)
```

golangci-lint (2.12.2)

[error] Error return value of `yml.SetOrAddNodeValue` is not checked

(errcheck)